package.json

```diff
@@ -18,7 +18,7 @@
 		"npm": "5.3.0"
 	},
 	"dependencies": {
-		"axios": "0.21.1",
+		"axios": "0.21.2",
```

Comment:
And similarly, we might as well bump to `0.24` unless something breaks.

Reply:
Let's go crazy!